standalone/src/test/java/io/pravega/local/TlsEnabledInProcPravegaClusterTest.java

```diff
@@ -77,7 +68,7 @@ public void testCreateStreamFailsWithInvalidClientConfig() {
 
     @Test(timeout = 30000)
     public void testWriteAndReadEventWithValidClientConfig() throws Exception {
-        testWriteAndReadAnEvent(scope, stream, msg, prepareValidClientConfig());
+        testWriteAndReadAnEvent(scope, stream, msg, EMULATOR.getClientConfig());
```

Comment:
If I remember correctly, using this static `EMULATOR` is not conflicting with PR #4999 right?

Reply:
No, it does not. PR #4999 does not execute @Test methods of a single Test file in parallel.